javascript-primitives-and-variables/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Javascript Primitives and Variables</title>
+</head>
+<body>
+  <script src="main.js">
+    var fullName = "Morris Wang";
+    console.log('full name:', fullName);
+    var isCool = true;
+    console.log('I am cool:', isCool);
+    var totalPets = 1;
+    console.log('total pets:', totalPets);
```

Comment:
a nice demo on embedded JS, but I would remove it as the same info is being already done in `main.js`